docs/en/stack/ml/setup.asciidoc

```diff
@@ -5,100 +5,140 @@
 <titleabbrev>Setup and security</titleabbrev>
 ++++
 
-To use the {stack} {ml-features}, you must have the 
-{subscriptions}[appropriate subscription] and at least one 
-<<ml-nodes,{ml} node>> in your cluster.
+[discrete]
+[[requirements-overview]]
+== Requirements overview
+
+To use the {stack} {ml-features}, you must have:
 
-If {stack} {security-features} are enabled, you must also ensure your users have
-the <<setup-privileges,necessary privileges>>. If the {operator-feature} is
-enabled, there are some {ml} settings that can be updated only by operator 
-users.
+[%interactive]
+- [ ] the {subscriptions}[appropriate subscription] level or the free trial 
+  period activated
+- [ ] `xpack.ml.enabled` set to its default value of `true` on every node in the 
+  cluster (refer to {ref}/ml-settings.html[{ml-cap} settings in {es}])
+- [ ] `ml` value defined in the list of `node.roles` on the 
+  {ref}/modules-node.html#ml-node[{ml} nodes]
+- [ ] {ml} features visible in the {kib} space
+- [ ] security privileges assigned to the user that:
+    * grant use of {ml-features}, and
+    * grant access to source and destination indices.
 
 TIP: The fastest way to get started with {ml-features} is to
 {ess-trial}[start a free 14-day trial of {ess}] in the cloud.
 
-[discrete]
-[[ml-nodes]]
-== Machine learning nodes
-
-To use {ml-features}, there must be at least one {ml} node in your cluster. A
-{ml} node is a node that has `xpack.ml.enabled` set to `true` and `ml` in
-`node.roles`.
-
-If nodes do not have the {ml} role, they cannot run {ml} jobs. If
-`xpack.ml.enabled` is `true`, however, they can service API requests. For more
-information, see {ref}/modules-node.html#ml-node[{ml-cap} nodes] and
-{ref}/ml-settings.html[{ml-cap} settings in {es}].
 
 [discrete]
 [[setup-privileges]]
 == Security privileges
 
+Security privileges can be assigned as either {es} privileges or {kib} feature 
+privileges. The user is granted the most permissive combination of both. You can 
```

Comment:
Nit picking here:
`The user is granted the most permissive combination of both.` is only true when using Kibana (non dev tools). We're restricting it two sentences later: `If you use {ml-features} via {es} APIs [...]`, but I'm wondering if we want to slightly modify this initial statement to make it crystal clear.